engine/src/cubos/engine/gizmos/renderer.cpp

```diff
@@ -1,12 +1,35 @@
 #include "renderer.hpp"
+#include <string>
 
 using cubos::engine::GizmosRenderer;
 
 using namespace cubos::core::gl;
 
-void GizmosRenderer::init(RenderDevice* currentRenderDevice)
+void GizmosRenderer::initIdTexture(glm::ivec2 size)
+{
+    cubos::core::gl::Texture2DDesc texDesc;
```

Comment:
Unnecessary namespace, and same below. You already have 'using namespace cubos::core::gl' at the top of the file.